lib/sles4sap/console_redirection.pm

```diff
@@ -37,6 +37,7 @@ our @EXPORT = qw(
   connect_target_to_serial
   disconnect_target_from_serial
   check_serial_redirection
+  $ssh_opt
```

Comment:
Could you tell something about ssh_opt . It was internal to console_redirection before. Why does it needs to be exposed now? I see it is used by [prepare_ssh_config.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21148/files#diff-7c32df9e81b786b4d90a800de5d5b1bc90e623771e6ad252d614a776c1d652b3) that is from an  SDAF specific lib.